test/e2e/conformance_test.go

```diff
@@ -134,6 +137,15 @@ func TestClusterConformance(t *testing.T) {
 			// Create infrastructure
 			t.Log("Provisioning infrastructure using Terraform…")
 			args := []string{}
+			img, err := DiscoverControlPlaneOSImage(tc.provider, testOperatingSystem)
```

Comment:
this `DiscoverControlPlaneOSImage()` forces custom AMIs, not sure if we should hardcode AMIs, maybe make them optional.

Reply:
Fixed, it's optional now.